app/jobs/tracking_sheet_report_job.rb

```diff
@@ -0,0 +1,142 @@
+      x: (pdf.bounds.width / 2) - (bc_width / 2),
+      y: (pdf.bounds.height - bc_height)
+    )
+
+    pdf.y -= (bc_height + 0.25.in)
+    pdf.text druid, size: 15, style: :bold, align: :center
+    pdf.y -= 0.5.in
+
+    pdf.font('Courier', size: 10)
+    pdf.table(doc_to_table(doc), column_widths: [100, 224], cell_style: { borders: [], padding: 0.pt })
+
+    pdf.y -= 0.5.in
+
+    pdf.font_size = 14
+    pdf.text 'Tracking:'
+    pdf.text ' '
+
+    baseline = pdf.y - top_margin - pdf.font.ascender
+    pdf.rectangle([0, baseline + pdf.font.ascender], pdf.font.ascender, pdf.font.ascender)
+    pdf.indent(pdf.font.ascender + 4.pt) do
+      pdf.text 'Scanned by:'
+      pdf.indent(pdf.width_of('Scanned by:') + 0.125.in) do
+        pdf.line 0, baseline, pdf.bounds.width, baseline
+      end
+    end
+    pdf.stroke
+
+    pdf.y -= 0.5.in
+    pdf.text('Notes:')
+    pdf.stroke do
+      while pdf.y >= pdf.bounds.absolute_bottom
+        baseline = pdf.y - top_margin - pdf.font.height
+        pdf.line 0, baseline, pdf.bounds.width, baseline
+        pdf.y -= pdf.font.height * 1.5
+      end
+    end
+    pdf
+  end
+
+  # @param [Hash] doc Solr document or to_solr Hash
+  # @return [Array<Array<String>>] Complex array suitable for pdf.table()
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
+  def doc_to_table(doc)
+    table_data = []
+    labels = doc[SolrDocument::FIELD_LABEL]
+    label = labels.blank? ? '' : labels.first
+    label = "#{label[0..110]}..." if label.length > 110
+    table_data.push(['Object Label:', label])
+    table_data.push(['Project Name:', doc['project_tag_ssim'].to_s]) if doc['project_tag_ssim']
+
+    tags = Array(doc['tag_ssim']).collect { |tag| /^Project\s*:/.match?(tag) ? nil : tag.gsub(/\s+/, Prawn::Text::NBSP) }.compact
+    table_data.push(['Tags:', tags.join("\n")]) unless tags.empty?
+    table_data.push(['Catkey:',    Array(doc[SolrDocument::FIELD_CATKEY_ID]).join(', ')]) if doc[SolrDocument::FIELD_CATKEY_ID].present?
+    table_data.push(['Source ID:', Array(doc['source_id_ssim']).first]) if doc['source_id_ssim'].present?
+    table_data.push(['Barcode:',   Array(doc['barcode_id_ssim']).first]) if doc['barcode_id_ssim'].present?
+    table_data.push(['Date Printed:', Time.zone.now.strftime('%c')])
+    table_data
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+  # rubocop:enable Metrics/PerceivedComplexity
+
+  # @param [String] druid unqualified DRUID identifier
+  # @return [Hash] doc from Solr or to_solr
+  # @note To the extent we use Solr input filters or copyField(s), the Solr version will differ from the to_solr hash.
+  # @note That difference shouldn't be important for the few known fields we use here.
+  def find_or_create_in_solr_by_id(druid)
+    namespaced_druid = Druid.new(druid).with_namespace
+    doc = SearchService.query(%(id:"#{namespaced_druid}"), rows: 1)['response']['docs'].first
+    return doc unless doc.nil?
```

Comment:
I don't understand this use case. Can you explain it to me?  Why wouldn't it be found in solr? Why is this the right place to fix it?

Reply:
This was a straight copy of the existing code used in the registration form, so I can't answer that. it is however 6 yo code (https://github.com/sul-dlss/argo/commit/ee0a7be508fde4c28d26dbb2e333f435420ab6b4)... if we'd like to refactor that, I'm happy to do so.

Reply:
I think that code makes sense in registration, when the object may have not been created, but I don't think we need it here.